phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexIT.java

```diff
@@ -45,19 +45,24 @@
 	private final boolean localIndex;
 	private final String tableDDLOptions;
 	
```

Comment:
Instead of using System.currentMillis() in the table name (which will create a new physical table for every test), can you create a TestUtil.getTableName(String prefix, boolean localIndex, boolean mutable, boolean transactional, Integer saltBuckets) and creates a sensible table name like T_LOCAL_MUTABLE_UNSALTED.